packages/payload/src/uploads/safeFetch.ts

```diff
@@ -57,13 +82,13 @@ export const safeFetch = async (...args: Parameters<typeof undiciFetch>) => {
         throw new Error(error.cause.message)
       } else {
         let stringifiedUrl: string | undefined | URL = undefined
-        if (typeof url === 'string' || url instanceof URL) {
-          stringifiedUrl = url
-        } else if (url instanceof Request) {
-          stringifiedUrl = url.url
+        if (typeof unverifiedUrl === 'string' || unverifiedUrl instanceof URL) {
```

Comment:
nitpick: to prevent needing to use `String()` at all later, I would've expected something more like this:

```ts
let stringifiedUrl: string | undefined = undefined
if (typeof unverifiedUrl === 'string') {
  stringifiedUrl = unverifiedUrl
} else if (unverifiedUrl instanceof URL) {
  stringifiedUrl = unverifiedUrl.toString()
} else if (unverifiedUrl instanceof Request) {
  stringifiedUrl = unverifiedUrl.url
}

throw new Error(`Failed to fetch from ${stringifiedUrl}, ${error.message}`)
```

Not necessary, but worth noting.